public/scripts/contollers/routes.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
```

Comment:
The routes listed in this object should match, exactly, the ones listed in index.html! There's a problem here.